ngrok-java-native/src/lib.rs

```diff
@@ -469,6 +469,13 @@ impl<'local> NativeSessionRsImpl<'local> {
 
         bldr.proxy_proto(ProxyProto::from(jeb.get_proxy_proto_version(self.env)));
 
+        if let Some(policy) = jeb.get_policy(self.env).of_string(self.env) {
+            match bldr.policy(policy.as_str()) {
```

Comment:
I think you can rewrite this as: 
```rust
            bldr.policy(policy.as_str()).map_err(io_exc_err)?;
```

Reply:
That gives me the following compilation error:
```
error[E0277]: `?` couldn't convert the error to `jaffi_support::Error<IOExceptionErr>`
   --> src/lib.rs:520:61
    |
520 |             bldr.policy(policy.as_str()).map_err(io_exc_err)?;
    |                                                             ^ the trait `From<Result<_, jaffi_support::Error<IOExceptionErr>>>` is not implemented for `jaffi_support::Error<IOExceptionErr>`
    |
    = note: the question mark operation (`?`) implicitly performs a conversion on the error value using the `From` trait
    = help: the following other types implement trait `FromResidual<R>`:
              <Result<T, F> as FromResidual<Result<Infallible, E>>>
              <Result<T, F> as FromResidual<Yeet<E>>>
    = note: required for `Result<TlsTunnelBuilder, jaffi_support::Error<IOExceptionErr>>` to implement `FromResidual<Result<Infallible, Result<_, jaffi_support::Error<IOExceptionErr>>>>`
```

Reply:
its `map_err(io_exc)?`, e.g. `io_exc` not `io_exc_err`.